src/encoded/types/file.py

```diff
@@ -37,6 +66,9 @@ class File(SMAHTItem, CoreFile):
 
     SHOW_UPLOAD_CREDENTIALS_STATUSES = ("in review",)
 
+    class Collection(SMAHTItem.Collection):
```

Comment:
Also, I think we should update line 67 above to appropriate statuses included in this PR, namely `uploading`, `to be uploaded by workflow`, and `upload failed` (not entirely sure about the latter, but these are the 3 we use in encoded-core). There's a corresponding test that will need to be updated and fixed if done here.

Reply:
This will come in an upcoming PR very soon, for now I'd like to isolate the status related changes